terraform/Makefile

```diff
@@ -0,0 +1,22 @@
+SHELL=/bin/bash
+REGION?="us-west-2"
+
+prepare:
+	$(eval AWS=$(shell echo pass hashbang/aws/terraform))
```

Comment:
@KellerFuchs I've been struggling with `Makefile` quite a bit to get this to work how I wanted (just `make plan`, press yubikey twice, and go), best case would be in a single GPG roundtrip, but after a few hours this is what I ended up with. If you have a better way to do this I'm all ears.